tests/toStringTests.java

```diff
@@ -0,0 +1,181 @@
+package tests;
+
+import com.laboon.*;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class toStringTests {
+	
+    // make sure the string returned by toString is not null
+    @Test
+    public void toStringNotNullTest() {
+        World world = new World(10, 1, 50);
+        String str = world.toString();
+        assertNotNull(str);
```

Comment:
Isn't this result implied if the next test passes?  These tests overlap somewhat.